docs/DesignLanguage.vue

```diff
@@ -300,6 +300,11 @@
                         {{ models.textInput0 }}
                         <code>{{ groups['input'].components[0].examples[0].code }}</code>
                     </div>
+                    <div class="example">
+                        <h5>Error State</h5>
+                        <ff-text-input placeholder="Insert something here..." v-model="models.textInput0" :error="'This is the error'"/>
```

Comment:
```suggestion
                        <ff-text-input placeholder="Insert something here..." v-model="models.textInput1" :error="'This is the error'"/>
```

Giving the error example input its own model.